qa/L0_trace/test.sh

```diff
@@ -698,68 +698,40 @@ set +e
 # Check opentelemetry trace exporter sends proper info.
 # A helper python script starts listening on $OTLP_PORT, where
 # OTLP exporter sends traces.
-export TRITON_OPENTELEMETRY_TEST='false'
 OTLP_PORT=10000
-OTEL_COLLECTOR_DIR=./opentelemetry-collector
-OTEL_COLLECTOR=./opentelemetry-collector/bin/otelcorecol_*
+OTEL_COLLECTOR=./otelcol
 OTEL_COLLECTOR_LOG="./trace_collector_http_exporter.log"
 
-# Building the latest version of the OpenTelemetry collector.
+# Installing OpenTelemetry collector (v0.91.0).
 # Ref: https://opentelemetry.io/docs/collector/getting-started/#local
-if [ -d "$OTEL_COLLECTOR_DIR" ]; then rm -Rf $OTEL_COLLECTOR_DIR; fi
-git clone --depth 1 --branch v0.82.0 https://github.com/open-telemetry/opentelemetry-collector.git
-cd $OTEL_COLLECTOR_DIR
-make install-tools
-make otelcorecol
-cd ..
-$OTEL_COLLECTOR --config ./trace-config.yaml >> $OTEL_COLLECTOR_LOG 2>&1 & COLLECTOR_PID=$!
-
-
-SERVER_ARGS="--trace-config=level=TIMESTAMPS --trace-config=rate=1 \
-                --trace-config=count=100 --trace-config=mode=opentelemetry \
-                --trace-config=opentelemetry,url=localhost:$OTLP_PORT/v1/traces \
-                --model-repository=$MODELSDIR"
-SERVER_LOG="./inference_server_otel_http_exporter.log"
-
-run_server
-if [ "$SERVER_PID" == "0" ]; then
-    echo -e "\n***\n*** Failed to start $SERVER\n***"
-    cat $SERVER_LOG
-    exit 1
-fi
-
-$SIMPLE_HTTP_CLIENT >>$CLIENT_LOG 2>&1
-
-set -e
-
-kill $SERVER_PID
-wait $SERVER_PID
-
-kill $COLLECTOR_PID
-wait $COLLECTOR_PID
-
-set +e
-
-if ! [[ -s $OTEL_COLLECTOR_LOG && `grep -c 'InstrumentationScope triton-server' $OTEL_COLLECTOR_LOG` == 3 ]] ; then
-    echo -e "\n***\n*** HTTP exporter test failed.\n***"
-    cat $OTEL_COLLECTOR_LOG
-    exit 1
-fi
-
+curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.91.0/otelcol_0.91.0_linux_amd64.tar.gz
+tar -xvf otelcol_0.91.0_linux_amd64.tar.gz
 
+rm collected_traces.json*
 # Unittests then check that produced spans have expected format and events
 OPENTELEMETRY_TEST=opentelemetry_unittest.py
 OPENTELEMETRY_LOG="opentelemetry_unittest.log"
-EXPECTED_NUM_TESTS="3"
-
-export TRITON_OPENTELEMETRY_TEST='true'
-
-SERVER_ARGS="--trace-config=level=TIMESTAMPS --trace-config=rate=1 \
-                --trace-config=count=100 --trace-config=mode=opentelemetry \
+EXPECTED_NUM_TESTS="13"
+
+# Set up repo and args for SageMaker
+export SAGEMAKER_TRITON_DEFAULT_MODEL_NAME="simple"
+MODEL_PATH="/opt/ml/models/123456789abcdefghi/model"
+rm -r ${MODEL_PATH}
+mkdir -p "${MODEL_PATH}"
+cp -r $DATADIR/$MODELBASE/* ${MODEL_PATH} && \
+    rm -r ${MODEL_PATH}/2 && rm -r ${MODEL_PATH}/3 && \
+        sed -i "s/onnx_int32_int32_int32/simple/" ${MODEL_PATH}/config.pbtxt
+
+
+SERVER_ARGS="--allow-sagemaker=true --model-control-mode=explicit \
+                --load-model=simple --load-model="ensemble_add_sub_int32_int32_int32" \
```

Comment:
Is this nested quote for ensemble model name OK?

Reply:
Yes, from recent local run:
```
I0111 22:47:57.913510 1734 server.cc:676]
+------------------------------------+---------+--------+
| Model                              | Version | Status |
+------------------------------------+---------+--------+
| bls_simple                         | 1       | READY  |
| ensemble_add_sub_int32_int32_int32 | 1       | READY  |
| simple                             | 1       | READY  |
+------------------------------------+---------+--------+
```
And from CI's logs:
```

I0111 19:03:11.859289 3243 server.cc:676] 
+------------------------------------+---------+--------+
| Model                              | Version | Status |
+------------------------------------+---------+--------+
| bls_simple                         | 1       | READY  |
| ensemble_add_sub_int32_int32_int32 | 1       | READY  |
| simple                             | 1       | READY  |
+------------------------------------+---------+--------+
```
It is getting loaded, but I'll remove quotes for consistency